### DISCUSSION TITLE
Why this over KDE Connect?

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [localsend/localsend#751] [Discussion (English, International)] [answered]
Why I should use LocalSend instead of KDE Connect to send my file to an Android device?

**Comment by Tienisto** [comment, marked as answer]
LocalSend is more lightweight, you can also share a link so others can download the file without needing to install LocalSend.

Both solutions are free and open source, so you are not forced to use one over another. Just use the solution you like the most :)

**Comment by jelabarre59** [comment]
KDE Connect does a lot of other things, not just file transfers.  On the other hand, KDE Connect can't transfer a directory, just individual files, while LocalSend can.  I use each for different purposes.

**Reply by blackshtef** [reply]
Same here. Not sure if Localsend has this feature - but KDE Connect has it - is the ability to control multimedia on your PC from phone, and if you get or start a call, it will automatically pause the multimedia on your PC and resume it afterwards.
